packages/cloud_firestore/lib/src/document_reference.dart

```diff
@@ -42,6 +42,17 @@ class DocumentReference {
     );
   }
 
+  Future<DocumentSnapshot> get() {
+    return Firestore.channel.invokeMethod('DocumentReference#get',
```

Comment:
I think this would be more idiomatic as an async method. You can use `await` instead of `then`

Reply:
Done!